diff --git a/views_bootstrap.info.yml b/views_bootstrap.info.yml
index ce5ca5f..1b77a7d 100644
--- a/views_bootstrap.info.yml
+++ b/views_bootstrap.info.yml
@@ -3,6 +3,7 @@ type: module
 description: 'Views Twitter Bootstrap Components.'
 package: Views
 core: 8.x
+core_version_requirement: ^8 || ^9
 dependencies:
   - drupal:views
 libraries:
diff --git a/views_bootstrap.theme.inc b/views_bootstrap.theme.inc
index a94d60d..cb4256c 100644
--- a/views_bootstrap.theme.inc
+++ b/views_bootstrap.theme.inc
@@ -31,16 +31,16 @@ function template_preprocess_views_bootstrap_accordion(array &$vars) {
     }
   }
   else {
-    // @TODO: This would be better as valdiation errors on the style plugin options form.
-    drupal_set_message(t('@style style will not display without the "@field" setting.',
-      [
+    // @TODO: This would be better as validation errors on the style plugin options form.
+    Drupal::messenger()->addWarning(
+      t('@style style will not display without the "@field" setting.', [
         '@style' => $view->style_plugin->definition['title'],
         '@field' => 'Panel title',
       ]
-    ), 'warning');
+    ));
   }
-  // @TODO: Make sure that $vars['rows'] is rendered array.
-  // @SEE: Have a look template_preprocess_views_view_unformatted()
+  // @TODO: Make sure that $vars['rows'] is a rendered array.
+  // @SEE: Have a look at template_preprocess_views_view_unformatted()
   // and views-view-unformatted.html.twig
 }
 
@@ -199,13 +199,13 @@ function template_preprocess_views_bootstrap_tab(array &$vars) {
     }
   }
   else {
-    // @TODO: This would be better as valdiation errors on the style plugin options form.
-    drupal_set_message(t('@style style will not display without the "@field" setting.',
-      [
+    // @TODO: This would be better as validation errors on the style plugin options form.
+    \Drupal::messenger()->addWarning(
+      t('@style style will not display without the "@field" setting.', [
         '@style' => $view->style_plugin->definition['title'],
         '@field' => 'Tab title',
       ]
-    ), 'warning');
+    ));
   }
 }
 
